src/sonic-config-engine/tests/test_multinpu_cfggen.py

```diff
@@ -158,6 +158,20 @@ def test_backend_asic_portchannel_intf(self):
         self.assertListEqual(output.keys(), \
             ['PortChannel4013', 'PortChannel4013|10.1.0.2/31', 'PortChannel4014', 'PortChannel4014|10.1.0.6/31'])
 
+    def test_frontend_asic_ports(self):
+        argument = "-m {} -p {} -n asic0 --var-json \"PORT\"".format(self.sample_graph, self.port_config[0])
+        output = json.loads(self.run_script(argument))
+        self.assertDictEqual(output, \
+           {
+            "Ethernet0": { "admin_status": "up",  "alias": "Ethernet1/1",  "asic_port_name": "Eth0-ASIC0",  "description": "01T2:Ethernet1",  "index": "0",  "lanes": "33,34,35,36",  "mtu": "9100",  "pfc_asym": "off",  "role": "Ext",  "speed": "40000" }, 
```

Comment:
Suggest to combine these lines to align with the other `self.assertDictEqual()` calls in this file.

Reply:
Fixed in the latest commit